x-pack/plugins/maps/ui_tests/README.md

```diff
@@ -0,0 +1,20 @@
+## How to run tests
```

Comment:
I think we want a code owners entry to make sure we own `x-pack/plugins/maps/ui_tests`. This reduces noise for the maps team while we're working on things and we'll transfer ownership once we're ready. We did the same for the discover UI tests.

Reply:
https://github.com/elastic/kibana/pull/204607/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R1582